### DISCUSSION TITLE
RAW data (units) format to get the .npz

### DISCUSSION THREAD

**Discussion by lucasem1** [mir-group/nequip#279] [General]
Dear Developers

I have a rich dataset made of Energies, Coordinates, Forces, and lattice parameter that I would like to use to train a FF.
Could you please show me which is the raw data format I have to pass to python for getting the .npz file? 
I need also to know which are the correct units of coords/energy/forces/lattice to pass to nequip for the training.
Best regards
Luca Sementa

**Comment by Linux-cpp-lisp** [comment]
Hi @lucasem1 ,

The prefered data input format is extxyz, which can be constructed and manipulated using the ASE library: https://wiki.fysik.dtu.dk/ase/, https://gist.github.com/simonbatzner/1806e8761d4f17a89285a6a84108eed5

`nequip` is agnostic to units and will predict, output errors, etc. in whatever units are given. The units just have to be consistant, i.e. if energy is in eV and positions are in Angstrom, then forces must be in eV / A.
